apps/web/src/lib/auth/authService.ts

```diff
@@ -1,19 +1,18 @@
-import { get, writable } from 'svelte/store';
+import { persisted } from '@gitbutler/shared/persisted';
+import { type Readable } from 'svelte/store';
 
 export class AuthService {
-	token = writable<string | undefined>(undefined);
+	#token = persisted<string | undefined>(undefined, 'AuthService--token');
```

Comment:
As discussed, it's not ideal that we store the token in local storage, but it is probably the best option until we're able to move to a HttpOnly cookie with expiring tokens.